tests/smoke/click.html

```diff
@@ -1,17 +1,17 @@
 <!doctype html>
 <html>
   <head>
-    <script src="../../bower_components/webcomponents-platform/webcomponents-platform.js"></script>
-    <script src="../../bower_components/template/template.js"></script>
+    <script src="../../node_modules/webcomponents-platform/webcomponents-platform.js"></script>
```

Comment:
Looks like `@webcomponents` scope is missing here.

Reply:
Good catch, thanks!